Python/compile.c

```diff
@@ -161,6 +161,9 @@ struct compiler {
     int c_optimize;              /* optimization level */
     int c_interactive;           /* true if in interactive mode */
     int c_nestlevel;
+    int c_emit_bytecode;         /* The compiler won't emmit any bytecode
+                                    if this flag is false. This can be used
+                                    to visit nodes to check only errors. */
```

Comment:
"This can be used to visit nodes to check only errors."

I would add "temporarily" somewhere :-) I don't think that a compiler should only start with c_emit_bytecode=1, no?

Reply:
Added in the last fixup!